htdocs/stripe/class/stripe.class.php

```diff
@@ -549,11 +543,10 @@ public function getPaymentIntent($amount, $currency_code, $tag, $description = '
 	 * @param	string 	$customer							Stripe customer ref 'cus_xxxxxxxxxxxxx' via customerStripe()
 	 * @param	string	$key							    ''=Use common API. If not '', it is the Stripe connect account 'acc_....' to use Stripe connect
 	 * @param	int		$status							    Status (0=test, 1=live)
-	 * @param	int		$usethirdpartyemailforreceiptemail	1=use thirdparty email for receipt
 	 * @param   boolean $confirmnow                         false=default, true=try to confirm immediatly after create (if conditions are ok)
 	 * @return 	\Stripe\SetupIntent|null 			        Stripe SetupIntent or null if not found and failed to create
 	 */
-	public function getSetupIntent($description, $object, $customer, $key, $status, $usethirdpartyemailforreceiptemail = 0, $confirmnow = false)
+	public function getSetupIntent($description = null, $object = null, $customer = null, $key = null, $status, $confirmnow = false)
```

Comment:
The 3 first parameters can be set to null as default value because $status (param4) has no default value and
"Arguments with default values must be at the end of the argument list"